src/stcal/jump/jump.py

```diff
@@ -864,19 +870,23 @@ def near_edge(jump, low_threshold, high_threshold):
 
 def find_faint_extended(
     indata,
-    gdq,
+    ingdq,
+    pdq,
     readnoise_2d,
     nframes,
     minimum_sigclip_groups,
     snr_threshold=1.3,
     min_shower_area=40,
     inner=1,
     outer=2,
+    donotuse_flag = 1,
     sat_flag=2,
     jump_flag=4,
+    refpix_flag=2147483648,
```

Comment:
In general it's bad form and we discourage the use of explicit/hardwired DQ flag values in the code. Should import the dqflags definitions and use those instead.

Reply:
I removed the hard coded DQ values for reference pixels.